spec/features/form/form_navigation_spec.rb

```diff
@@ -211,15 +211,15 @@
         fill_in("lettings-log-address-line1-input-field", with: "address")
         fill_in("lettings-log-postcode-full-input-field", with: "A1 1AA")
         click_button(text: "Search")
-        expect(page).to have_current_path("/lettings-logs/#{id}/uprn-selection?referrer=check_answers")
+        expect(page).to have_current_path("/lettings-logs/#{id}/uprn-selection?referrer=check_answers&unanswered_pages=property_local_authority")
```

Comment:
How is this changing the test?

Reply:
I'm not sure why my comment on this change is marked as outdated but it is here
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/2485#discussion_r1654941111

as there, it doesn't strike me, but I would appreciate any further context, that adding something to the query string is a problem. But I will look into it a little further